pkg/metrics/series_test.go

```diff
@@ -72,7 +72,7 @@ func TestPopulateDeviceField(t *testing.T) {
 		},
 		{
 			[]string{"some:tag", "device_name:/dev/sda1"},
-			[]string{"some:tag"},
+			[]string{"some:tag", "device_name:/dev/sda1"},
 			"/dev/sda1",
 		},
```

Comment:
I'm going to add a test to verify that `device_name` does not overwrite the `device` value.